src/controllers/auth.controller.js

```diff
@@ -0,0 +1,24 @@
+const authService = require("../services/auth.service");
+
+const signUp = async (req, res) => {
+  const { name, dateOfBirth, phoneNumber, gender, address, email, password } =
+    req.body;
+  await authService.signUp(
+    name,
+    dateOfBirth,
+    phoneNumber,
+    gender,
+    address,
+    email,
+    password
+  );
+  res.status(201).end();
```

Comment:
데이터가 생성된 결과를 statusCode로 전달해도 충분하겠지만, 클라이언트에게 명확한 메시지를 던져도 좋을 것 같습니다!

```suggestion
  res.status(201).json({ message: "success" });
```